### PR TITLE
added  monsters

### PR DESCRIPTION
**Pull request by amytnyk**
please look at my new code, dad

### CODE REVIEW

File: SmallLabySfml/ClientModel.cpp [hunk 1]
```diff
@@ -2,16 +2,17 @@
...
+//using namespace SmallLabyUser::ServiceRefSmallLaby;
 
 struct ClientModel::PrivateModel
 {
-  gcroot<SmallLabyClient^> client;
+  gcroot<ServiceRefSmallLaby::SmallLabyClient^> client;
   PrivateModel()
   {
-    client = gcnew SmallLabyClient();
+    client = gcnew ServiceRefSmallLaby::SmallLabyClient();
```
Comment (mytnyk) [inline SmallLabySfml/ClientModel.cpp:15]: well.. not bad

Comment (amytnyk) [reply SmallLabySfml/ClientModel.cpp:15]: yea. it's really good

Comment (amytnyk) [reply SmallLabySfml/ClientModel.cpp:15]: yea. it's really good

Comment (amytnyk) [reply SmallLabySfml/ClientModel.cpp:15]: yea. it's really good

Comment (amytnyk) [reply SmallLabySfml/ClientModel.cpp:15]: yea. it's really good

**Pull request merged**